source/content/guides/drush/02-drush-versions.md

```diff
@@ -27,6 +27,12 @@ Pantheon runs Drush versions based on your Drupal version:
 
 We recommend managing your site through Composer. Visit the [Build Tools Workflow](/guides/build-tools/) for information on how to use Composer to manage Drupal sites on Pantheon, or the [Convert to Composer](/guides/composer-convert) guide to convert an existing site to a Composer-managed site.
 
+<Alert title="Note"  type="info" >
+
+Drush 11 is only available via Composer. We only support this version, Drush 11 has to be installed as a site-local Drush.
```

Comment:
Drupal 8 (no longer supported) starts with Drush 10: https://github.com/pantheon-systems/drops-8/blob/default/pantheon.upstream.yml#L10. Note there is a comment about "Drush 10" above that only applies to Drupal 8.

Drupal 9 & 10 add Drush via Composer. See https://www.drush.org./12.x/install/ for supported Drush versions, and what versions of Drupal they work with.